bin/run-tests-in-docker.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Synopsis:
+# Test the analyzer Docker image by running it against a predefined set of 
+# solutions with an expected output.
+# The analyzer Docker image is built automatically.
+
+# Output:
+# Outputs the diff of the expected test results against the actual test results
+# generated by the test runner Docker image.
+
+# Example:
+# ./bin/run-tests-in-docker.sh
+
+set -e # Make script exit when a command fail.
```

Comment:
👍 (sanity check: the script fails when either the first or the second stage of the docker build fails, which was the problem Erik wanted to address)